[2025-02-02T19:12:23.000+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T19:12:23.013+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: save_to_minio.save_to_minio scheduled__2025-01-27T00:00:00+00:00 [queued]>
[2025-02-02T19:12:23.019+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: save_to_minio.save_to_minio scheduled__2025-01-27T00:00:00+00:00 [queued]>
[2025-02-02T19:12:23.020+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 8
[2025-02-02T19:12:23.028+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): save_to_minio> on 2025-01-27 00:00:00+00:00
[2025-02-02T19:12:23.032+0000] {standard_task_runner.py:72} INFO - Started process 5093 to run task
[2025-02-02T19:12:23.034+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'save_to_minio', 'save_to_minio', 'scheduled__2025-01-27T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/main_pipeline.py', '--cfg-path', '/tmp/tmpq13oh11b']
[2025-02-02T19:12:23.036+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask save_to_minio
[2025-02-02T19:12:23.070+0000] {task_command.py:467} INFO - Running <TaskInstance: save_to_minio.save_to_minio scheduled__2025-01-27T00:00:00+00:00 [running]> on host 15904aeddad7
[2025-02-02T19:12:23.129+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='andrewsis' AIRFLOW_CTX_DAG_ID='save_to_minio' AIRFLOW_CTX_TASK_ID='save_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-01-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-27T00:00:00+00:00'
[2025-02-02T19:12:23.130+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T19:12:26.263+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2025-02-02T19:12:26.264+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minio_conn', conn_type='aws') credentials retrieved from login and password.
[2025-02-02T19:12:26.647+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T19:12:26.655+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T19:12:26.656+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=save_to_minio, task_id=save_to_minio, run_id=scheduled__2025-01-27T00:00:00+00:00, execution_date=20250127T000000, start_date=20250202T191223, end_date=20250202T191226
[2025-02-02T19:12:26.700+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T19:12:26.720+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-02T19:12:26.722+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
